airflow-core/tests/unit/models/test_serialized_dag.py

```diff
@@ -386,6 +387,28 @@ def test_get_dependencies(self, session):
         dependencies = SDM.get_dag_dependencies(session=session)
         assert dag_id not in dependencies
 
+    def test_get_dependencies_with_asset_ref(self, dag_maker, session):
```

Comment:
Is this test related?

Reply:
yes, this tests whether dag_dep can correctly get on both compressed and non-compressed cases. (all the tests in this class runs in two mode)

Reply:
Ahhh 👍🏻